sysroot/Dockerfile_ubuntu_arm

```diff
@@ -65,8 +65,7 @@ RUN python3 -m pip install -U \
 
 RUN apt install --no-install-recommends -y \
     libasio-dev \
-    libtinyxml2-dev \
-    libssl-dev
```

Comment:
@lmayencourt How come `libssl-dev` got removed? My guess is the instructions won't work since they use `SECURITY=ON`...

cc @left4taco